package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "express-gateway",
+  "name": "@mld/express-gateway",
   "version": "1.15.0",
```

Comment:
Should we use some way of showing our forked version such as using "1.15.0.0" and bumping the last digit, or having a  "version" and a  "parent version" or something?